src/overlays/actors/ovl_En_Fall2/z_en_fall2.c

```diff
@@ -28,26 +33,136 @@ const ActorInit En_Fall2_InitVars = {
     (ActorFunc)EnFall2_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_06005EF4;
-extern UNK_TYPE D_06005F10;
-extern UNK_TYPE D_06008840;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/EnFall2_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/EnFall2_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/func_80C1B724.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/func_80C1B8A4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/func_80C1B8B4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/func_80C1B8F0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/func_80C1B9D4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/EnFall2_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Fall2/EnFall2_Draw.s")
+void EnFall2_Init(Actor* thisx, PlayState* play) {
+    EnFall2* this = THIS;
+
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = EnFall2_DoNothing;
+    func_80183430(&this->skeletonInfo, object_fall2_Blob_008898, object_fall2_Blob_005EF4, this->unk174, this->unk228,
+                  NULL);
+    func_801835EC(&this->skeletonInfo, object_fall2_Blob_005EF4);
+    this->unk2DC = Lib_SegmentedToVirtual(object_fall2_Matanimheader_008840);
+    Actor_SetScale(&this->actor, 0.02f);
+    this->actionFunc = func_80C1B9D4;
+    this->csActionIndex = 0x231;
+    this->alphaLevel = 0.0f;
+}
+
+void EnFall2_Destroy(Actor* thisx, PlayState* play) {
+    EnFall2* this = THIS;
+
+    func_8018349C(&this->skeletonInfo);
+}
+
+u8 sAlphaTableIndices[] = { 0, 2, 0, 0, 4, 0, 4, 4, 2, 0, 0, 0, 0, 0, 0, 2, 4, 4, 4, 2, 0, 4, 2, 2, 0, 4, 4, 4, 4, 0,
+                            2, 2, 4, 0, 2, 0, 4, 4, 2, 0, 2, 4, 4, 4, 2, 0, 4, 4, 4, 2, 2, 0, 4, 4, 2, 4, 0, 2, 0, 0,
+                            2, 4, 4, 2, 0, 2, 4, 4, 4, 2, 0, 4, 4, 4, 4, 4, 2, 2, 0, 2, 4, 0, 2, 0, 0, 2, 4, 4, 2, 0,
+                            2, 4, 4, 4, 2, 0, 0, 2, 2, 4, 4, 4, 4, 4, 4, 2, 0, 2, 0, 0, 2, 4, 4, 2, 0, 2, 4, 4, 4, 2,
+                            0, 2, 0, 5, 2, 5, 5, 5, 5, 0, 2, 5, 0, 2, 0, 2, 5, 5, 2, 0, 2, 5, 5, 5, 2, 0, 3, 3, 3, 2,
+                            2, 0, 0, 3, 3, 0, 2, 2, 3, 0, 2, 3, 3, 2, 0, 2, 3, 3, 3, 2, 0, 2, 2, 2, 2, 0, 2, 2, 2, 2,
+                            2, 0, 0, 2, 0, 2, 2, 2, 2, 0, 2, 2, 2, 2, 2, 0, 1, 1, 0, 0, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1,
+                            1, 1, 0, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
```suggestion
u8 sAlphaTableIndices[] = { 0, 2, 0, 0, 4, 0, 4, 4, 2, 0, 0, 0, 0, 0, 0, 2, 4, 4, 4, 2, 0, 4, 2, 2, 0, 4, 4, 4, 4, 0,
                            2, 2, 4, 0, 2, 0, 4, 4, 2, 0, 2, 4, 4, 4, 2, 0, 4, 4, 4, 2, 2, 0, 4, 4, 2, 4, 0, 2, 0, 0,
                            2, 4, 4, 2, 0, 2, 4, 4, 4, 2, 0, 4, 4, 4, 4, 4, 2, 2, 0, 2, 4, 0, 2, 0, 0, 2, 4, 4, 2, 0,
                            2, 4, 4, 4, 2, 0, 0, 2, 2, 4, 4, 4, 4, 4, 4, 2, 0, 2, 0, 0, 2, 4, 4, 2, 0, 2, 4, 4, 4, 2,
                            0, 2, 0, 5, 2, 5, 5, 5, 5, 0, 2, 5, 0, 2, 0, 2, 5, 5, 2, 0, 2, 5, 5, 5, 2, 0, 3, 3, 3, 2,
                            2, 0, 0, 3, 3, 0, 2, 2, 3, 0, 2, 3, 3, 2, 0, 2, 3, 3, 3, 2, 0, 2, 2, 2, 2, 0, 2, 2, 2, 2,
                            2, 0, 0, 2, 0, 2, 2, 2, 2, 0, 2, 2, 2, 2, 2, 0, 1, 1, 0, 0, 1, 1, 1, 0, 1, 1, 1, 1, 1, 1,
                            1, 1, 0, 1, 0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, };
```
This should format better with a trailing comma.